run.py

```diff
@@ -170,14 +171,27 @@ def run_simu(n_stores=None, n_people=None, n_parks=None, n_misc=None,
         ) for i in range(n_misc)
     ]
 
+    new_dict = {}
+    for i in households:
+        new_dict[i.name] = 0, i.capacity
+
+    def _assign_household(rng):
+        household = rng.choice(households)
+        occ, cap = new_dict[household.name]
+        if (occ < cap):
+            new_dict[household.name] = occ + 1, cap
+        else:
+            _assign_household(rng)
+        return household
```

Comment:
Given that it is not taking  care of age, how does it ensure that 2 2year olds are not housed in a house of cap 2? 

I might be missing something here. Sorry about that!

Reply:
This PR is for Realistic People/House Ratio (Task - 89), Will submit a separate PR for the age distribution problem shortly.

Reply:
Hi Prateek, Modified taking age into consideration as well as people/house ratio.  Can you once have a look into it?